packit_service/worker/testing_farm.py

```diff
@@ -143,10 +143,29 @@ def _payload(
         if self.fmf_ref:
             fmf["ref"] = self.fmf_ref
 
+        predefined_environment = {
+            "full_repo_name": self.project.full_repo_name,
+            "upstream_name": self.job_config.upstream_package_name,
+            "upstream_url": self.job_config.upstream_project_url,
+            "downstream_name": self.job_config.downstream_package_name,
+            "downstream_url": self.job_config.downstream_project_url
+            if self.job_config.downstream_package_name
+            else None,
+            "commit_sha": self.metadata.commit_sha,
+        }
```

Comment:
I would use capital letters and maybe a prefix like `PACKIT_`. What do you think?